### PR TITLE
FrontName example link added on Routing topic

### PR DESCRIPTION
**Pull request by rajesh-mage**
## Purpose of this pull request

This pull request (PR) FrontName example link added on the Routing topic

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/extension-dev-guide/routing.html

### CODE REVIEW

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
This is good but please can you add this to the 2.3 version of the file. This can be done on this PR.

**Comment by rajesh-mage** [comment]
> This is good but please can you add this to the 2.3 version of the file. This can be done on this PR.

@BarnyShergold PR changes added on the 2.3 version. Kindly review and let me know If any changes.

Thanks.

**Comment by atwixfirster** [comment]
Hi, @rajesh-mage !

Honestly, I do not understand where did you get `routesxml` and what is a purpose of this PR?

Example for the `frontName` is provided on [line 229](https://github.com/magento/devdocs/pull/8934/files#diff-48752f91dd12ac1bfae8e6f7413e77590d303cf07ca1ad6b549ddcd31e9e3a2cR229).

CC: @BarnyShergold

**Comment by rajesh-mage** [comment]
Hi @atwixfirster 
> Hi, @rajesh-mage !
> 
> Honestly, I do not understand where did you get `routesxml` and what is a purpose of this PR?
> 
> Example for the `frontName` is provided on [line 229](https://github.com/magento/devdocs/pull/8934/files#diff-48752f91dd12ac1bfae8e6f7413e77590d303cf07ca1ad6b549ddcd31e9e3a2cR229).

In this PR I have added an example hyperlink to the routes.xml heading on the line of 97. So the user easily knows about the FrontName from the routes.xml heading section.

CC: @BarnyShergold

Thanks.

File: src/guides/v2.4/extension-dev-guide/routing.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Where:
    Example: [`Magento\Robots\Controller\Router`]
 *  `%sortorder%` - The sort order of this entry in the router list.
 
-## `routes.xml`
+## `routes.xml`
```
Comment (atwixfirster) [inline src/guides/v2.4/extension-dev-guide/routing.md:97]: Please remove a redundant SPACE

**Review by atwixfirster — CHANGES_REQUESTED** [review]

File: src/guides/v2.4/extension-dev-guide/routing.md (hunk 1 above)
Comment (rajesh-mage) [reply src/guides/v2.4/extension-dev-guide/routing.md]: @atwixfirster 
Space removed.

Thanks.

**Review by BarnyShergold — APPROVED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**